### PR TITLE
[.Net] Support image input for Anthropic Models

### PR DESCRIPTION
**Pull request by DavidLuong98**
## Why are these changes needed?
We want to be able to send images to the Anthropic models.

This PR makes it possible to send Images to Anthropic Models

## Related issue number
#2770 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [X] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [X] I've made sure all auto checks have passed.

### CODE REVIEW

File: dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs [hunk 1]
```diff
@@ -28,4 +31,32 @@ public async Task AnthropicAgentChatCompletionTestAsync()
...
+    public async Task AnthropicAgentTestProcessImageAsync()
+    {
+        var client = new AnthropicClient(new HttpClient(), AnthropicConstants.Endpoint, AnthropicTestUtils.ApiKey);
+        var agent = new AnthropicClientAgent(
+            client,
+            name: "AnthropicAgent",
+            AnthropicConstants.Claude3Haiku).RegisterMessageConnector();
+
+        var base64Image = await AnthropicTestUtils.Base64FromImageAsync("square.png");
+        var imageMessage = new ChatMessage("user",
```
Comment (LittleLittleCloud) [inline dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs:45]: Can you add tests for the following message types
- ImageMessage
- MultiModaMessage with one ImageMessage

Comment (LittleLittleCloud) [reply dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs:45]: You can refer to this code on how to do it
- https://github.com/microsoft/autogen/blob/1e5c03f66a0df121642acb277db3e72f52c8a318/dotnet/test/AutoGen.Ollama.Tests/OllamaAgentTests.cs#L117

**Comment by LittleLittleCloud** [comment]
Also, please install git lfs and put image.png under lfs tracking, otherwise the git lfs check will fail

File: dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs (hunk 1 above)
Comment (DavidLuong98) [reply dotnet/test/AutoGen.Anthropic.Tests/AnthropicClientAgentTest.cs:45]: Updated the code, thanks for the sample !

**Review by LittleLittleCloud — APPROVED** [review]

**Pull request merged**